### DISCUSSION TITLE
Microsoft Search for showing List Items which have specific content type, is not showing any property

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#3917] [Q&A]
I have added a Search Result, with those settings: -

![image](https://github.com/user-attachments/assets/3a63a1b8-55ba-4cfc-9be5-a96ae3d687f3)

And i added those slots which are linked to Search managed properties:-

![image](https://github.com/user-attachments/assets/83cd21ef-f39f-4d4f-891c-c8e67f4c9cf8)

and i added those to the Detailed list, as follow:-

![image](https://github.com/user-attachments/assets/68d180cc-622a-40b0-87c9-97fc8e5a4918)

![image](https://github.com/user-attachments/assets/72ceee0e-f0c6-4473-8a93-4f07fe4b5e87)

here is a sample of one expression:-

{{slot item @root.slots.SalesOrderNo}}

now i got the items, but all the custom property are empty as follow:-

![image](https://github.com/user-attachments/assets/63c792d1-7ccb-4648-8621-03d60b5d965e)

second question, how i can allow sorting the fields i have? such as Destination, Po# ? etc, is this possible?
Keeping in mind that i added the related data 3 days ago
any advice?

**Comment by kasperbolarsen** [comment]
In general, I will recommend staying away from Microsoft Search as a data source unless you are deal with external items (Graph Connector stuff ) . In my opinion MS Search is not ready for prime time yet.
Is your content in SharePoint ?

**Reply by mvcsharepointdev** [reply]
@kasperbolarsen yes they are inside custom lists inside sharepoint, i chose Microsoft Search as i belive they are more future proof, am i correctT?

**Reply by kasperbolarsen** [reply]
Unfortunately not, as Microsoft Search has been in hibernation ever since Copilot was announced.
I hope and expect that we will see some development later this year,  but for now I will recommend to wait as MS Search just isn't mature enough for serious work.

**Comment by wobba** [comment]
You need to add the properties you want into the selected properties list as well. And you can use the debug template to see what is returned per item.
